captum/optim/_core/loss.py

```diff
@@ -201,6 +209,15 @@ def __call__(self, targets_to_values: ModuleOutputMapping) -> torch.Tensor:
 class ChannelActivation(BaseLoss):
     """
     Maximize activations at the target layer and target channel.
+    This loss maximizes the activations of a target channel in a specified target
+    layer, and can be useful to determine what features the channel is excited by.
+
+    Args:
+        target (nn.Module):  The layer to containing the channel to optimize for.
+        channel_index (int):  The index of the channel to optimize for.
+        batch_index (int, optional):  The index of the image to optimize if we
+            optimizing a batch of images. If unspecified, defaults to all images
+            in the batch.

```

Comment:
I'm not sure if we should be calling the activations `images` in the batch index docs, as the activations can be images, or 2D / 4D activations.

Reply:
There's also a grammatical error in the batch_index docs: "optimize if we optimizing a batch of", should be "optimize if we **are** optimizing a batch of".